### PR TITLE
Add deal creation improvements

### PR DESCRIPTION
**Pull request by saeta-eth**

### CODE REVIEW

File: pages/deal/create.tsx [hunk 1]
```diff
@@ -98,6 +99,38 @@ const Create: NextPage = () => {
...
 
+  const handleConfirm = (
+    whitelist: Array<AddressWhitelistProps | NftCollectionRulesProps>,
+    type: NftType | string,
+  ) => {
+    setDealField(whitelist, type)
+    setDealField(true, 'withMerkleTree')
+
+    /*
+      TODO: we need to review the private pool logic to enable this code again.
```
Comment (0xDmitry) [inline pages/deal/create.tsx:110]: You gonna do it in next PR, right?

Comment (0xDmitry) [reply pages/deal/create.tsx:110]: Oh, it's an old code, lol

File: src/components/pools/deal/DealInformation.tsx [hunk 2]
```diff
@@ -96,8 +96,8 @@ export const DealInformation: React.FC<{
           title="Exchange rates"
           tooltip="The ratio at which investment tokens deposited in the pool will be exchanged for deal tokens"
         >
-          <Value>{`${deal.exchangeRates.investmentPerDeal.formatted} ${deal.underlyingToken.symbol} per ${pool.investmentTokenSymbol}`}</Value>
-          <Value>{`${deal.exchangeRates.dealPerInvestment.formatted} ${pool.investmentTokenSymbol} per ${deal.underlyingToken.symbol}`}</Value>
+          <Value>{`${deal.exchangeRates.dealPerInvestment.formatted} ${deal.underlyingToken.symbol} per ${pool.investmentTokenSymbol}`}</Value>
```
Comment (0xDmitry) [inline src/components/pools/deal/DealInformation.tsx:99]: Maybe add TODO to not forget to switch it again when we will fix origin bug in subgraph?

**Review by 0xDmitry — APPROVED** [review]

File: src/components/pools/deal/DealInformation.tsx (hunk 2 above)
Comment (saeta-eth) [reply src/components/pools/deal/DealInformation.tsx:99]: I think the subgraph is okay, the issue was the order

Comment (0xDmitry) [reply src/components/pools/deal/DealInformation.tsx:99]: Ah, I see, these variable names are little bit confusing 😅

**Review by 0xDmitry — APPROVED** [review]

**Pull request merged**